docs/DeveloperGuide.md

```diff
@@ -598,7 +598,36 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case ends.
 
-**Use case: UC11 - Clearing Address or Wedding Book**
+**Use case: UC11 - Viewing a wedding**
```

Comment:
Perhaps this is the same is UC05?

Reply:
Maybe you can change the name to "Viewing a Wedding" to reduce ambiguity of what the use case represents

Reply:
The name of UC05*